src/libYARP_init/src/CustomInit.cpp

```diff
@@ -48,6 +62,10 @@ yarp::os::Network::~Network() {
 void yarp::os::Network::init() {
     if (__custom_yarp_is_initialized==0) {
         initMinimum();
+        // If we init the clock inside the initMinum, it will loop into itself twice
+        // calling again the initMinimun, ending with __yarp_is_initialized counter 
+        // increased twice.
+        yarpClockInit();
```

Comment:
:warning: :-1::-1::-1: This means that wherever `initMinimum()` is called the clock is not initialized, and that's very wrong, `initMinimum()` means init yarp without static plugins...